portfolio/src/main/webapp/travel.html

```diff
@@ -17,28 +17,37 @@ <h1> Klaudia's Portfolio</h1>
       </div>
     </nav>
     <div id="content" >
+      <div class="section" class="welcome-section">
+      <img src="images/plane.jpg" alt="Image of a Plane" class="intro-image">
+      <h1> Klaudia's Travel Corner</h1>
       <p> I get bored of many things quite easily, but Travel isn't one of them! </p>
       <p> If I could spend all my time travelling, believe me I would, but unfortunately travel has huge costs associated with it and I'm not a millionaire <em> just yet!</em></p>
       <p>Below you can find some of the places I have travelled to, <strong> Enjoy! </strong> </p>
+      </div>
       <div class="travel-portfolio">
         <div class="gallery">
           <figure class="gallery__photo gallery__item--1">
-              <img src="images/image-1.jpg" alt="Photo of Venice " class="gallery__img">
+              <img src="images/image-1.jpg" alt="Photo of Venice " title="Venice" class="gallery__img">
           </figure>
           <figure class="gallery__photo gallery__item--2">
               <img src="images/image-2.jpg" alt="Photo of Klaudia in Salou" class="gallery__img">
+              <!-- <div class="hidden-text">John Doe</div> -->
```

Comment:
Do you need this comment? https://google.github.io/styleguide/htmlcssguide.html#Comments

Reply:
Same for all the comments below.

Reply:
I was trialing a hover effect that didn't work and forgot to remove these. will do now